core/src/main/resources/jenkins/console/ConsoleUrlProviderGlobalConfiguration/config.jelly

```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <j:if test="${h.hasPermission(app.ADMINISTER) and descriptor.enabled}">
```

Comment:
why is this permission check needed it's a global configuration value?

This prevents System read users from even seeing the config which seems wrong

Reply:
Copied from elsewhere. I will check this.

Reply:
Removed in [5f051c9](https://github.com/jenkinsci/jenkins/pull/8321/commits/5f051c9b0faf10e39f48122389f38b05ce2b3ef6).